filters/ignore_new_executors.cpp

```diff
@@ -0,0 +1,60 @@
+#include "glog/logging.h"
+
+#include "filters/ignore_new_executors.hpp"
+
+#include "stout/try.hpp"
+#include "stout/nothing.hpp"
+
+namespace mesos {
+namespace serenity {
+
+Try<Nothing> IgnoreNewExecutorsFilter::consume(const ResourceUsage &usage) {
+  std::unique_ptr<ExecutorMap<time_t>> newExecutorTimestamps =
+    std::unique_ptr<ExecutorMap<time_t>>(new ExecutorMap<time_t>());
+  ResourceUsage newUsage;
```

Comment:
It would be useful to use the same name of the variable as in other filters... I used "product" (: It's not necessary, but it would be good to change every filter for name "newUsage" or change here to "product" for consistency and readability. 

Secondly, I like the idea of making shared, mutable ResourceUsage to justmodify& pass them... not copying all the time (: